demo/sync/syncServices.js

```diff
@@ -1,8 +1,17 @@
 var config = require('./config.json');
 var _ = require('lodash');
-angular.module('wfm.sync',[]).service('syncService', ['$http', '$window', 'syncPool', 'userService', function($http, $window, syncPool, userService) {
-  return userService.getProfile($http, $window)
-    .then(syncPool.syncManagerMap);
+angular.module('wfm.sync',[]).service('syncService', ['$http', '$window', 'syncPool', 'Auth', function($http, $window, syncPool, Auth) {
+  if (Auth.keycloak) {
+    // retrieve the users profile from keycloak
+    return Auth.keycloak.loadUserProfile()
+      .success(syncPool.syncManagerMap)
```

Comment:
You will need to provide facade for the user id and use method. Then passport and keycloak will provide implementations for that method.